### DISCUSSION TITLE
Card Saldo

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1151] [Proposals]
### Goal

Novo Card para Assinaturas - Fatura Saldo.

### Context

Para criar o componente da área do Fatura Saldo, foi utilizado o componente Data Card como base, e através do recurso de "componentização" do slot para adicionar as informações necessárias.

### Impact

Proposta Work & Co - Novo App.

### Discussion Questions

Esse novo componente ficaria em qual biblioteca? Qual seria o time responsável?

### Do you have a visual proposal?

https://www.figma.com/file/seiNjy7YmPWEY3BfY4WYpp/Novos-componentes---Sprint-1?node-id=32%3A30380&t=odJ2DBpIItDwEY1S-1

**Comment by yceballost** [comment]
Las discussion deben ser sobre componentes sin lógica de producto. Esta discussion debería estar junto a la discussion que se abra de esos "slot components"
